docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+    mongodb:
+        extends:
+            file: docker-compose.yml
+            service: mongodb
+        mem_limit: 4g
+        labels:
+            - triton.cns.services=mongodb
+        network_mode: bridge
```

Comment:
If we're configuring the `network_mode` here then we're assuming a non-Triton deployment. Which is fine (it should be portable if possible), but:
-  If we're deploying to multiple hosts on a swarm cluster then if we're configuring `bridge` here we end up with NAT, which doesn't work for our service discovery pattern.
- If we're deploying on a single Docker host (which, fine for testing but not prod) then we don't need to configure this at all.